src/autowiring/marshaller.h

```diff
@@ -171,104 +153,66 @@ namespace autowiring {
   };
 
   template<typename T>
+  struct float_converter;
+
+#ifdef MSVC
```

Comment:
I think the macro you are looking for is `_MSC_VER`. `MSVC` isn't normally defined.

If it's working despite this bug, maybe all you need is `snprintf()` for all platforms. But on the other hand, Travis doesn't test with Visual Studio.